Bylaws.md

```diff
@@ -40,6 +40,12 @@ An antitrust compliance statement shall be read at the start of all Forum Meetin
     
  5. The allocation of customers, territories, or products in any way."
 
+At the start of a Forum, CWG or sub-group Teleconference, the following "Note Well" shall be read:
+
+"All participants are reminded that they must comply with the CA/Browser Forum anti-trust policy, code of conduct, and intellectual property rights agreement, which can be found in section 1.3 of the Bylaws and cabforum.org. Please contact the chair with any comments or concerns about these policies".
```

Comment:
This particular "Note Well" was discussed and accepted via the Member's Mailing List. I think the existing language is clear and makes sense but I will defer to Native English speakers for possible improvements.

Reply:
"All participants are reminded that they must comply with the CA/Browser Forum's Bylaws, which include an antitrust policy, a code of conduct, and an intellectual property rights agreement. Please contact the Forum Chair with any comments or concerns about the Bylaws or these policies."

Reply:
@

> "All participants are reminded that they must comply with the CA/Browser Forum's Bylaws, which include an antitrust policy, a code of conduct, and an intellectual property rights agreement. Please contact the Forum Chair with any comments or concerns about the Bylaws or these policies."

Ben, I think @clintwilson wanted to have a more specific reference. I'm happy without one but I'll defer to Clint because I'm not sure if this is a strong request from Apple's side.

Reply:
Resolving this thread as the proposed language seems to work for everyone.